client/lib/analytics/ad-tracking.js

```diff
@@ -110,6 +106,9 @@ const FACEBOOK_TRACKING_SCRIPT_URL = 'https://connect.facebook.net/en_US/fbevent
 		quoraPixelId: '420845cb70e444938cf0728887a74ca1',
 		outbrainAdvId: '00f0f5287433c2851cc0cb917c7ff0465e',
 		nanigansAppId: '653793',
+		donutsGtag: 'DC-8907854',
+		wpcomGoogleAdsGtag: 'AW-1067250390',
+		jetpackGoogleAdsGtag: 'AW-937115306',
```

Comment:
These seem to be the same, albeit, unprefixed IDs that we have here:

https://github.com/Automattic/wp-calypso/blob/dfe7904564d904f206a526bdb8409aee76d25338/config/desktop.json#L144-L145